client/src/unifyfs-sysio.c

```diff
@@ -998,12 +997,13 @@ ssize_t UNIFYFS_WRAP(read)(int fd, void* buf, size_t count)
 /* TODO: find right place to msync spillover mapping */
 ssize_t UNIFYFS_WRAP(write)(int fd, const void* buf, size_t count)
 {
-    LOGDBG("write was called for fd %d", fd);
     size_t ret;
     off_t pos;
 
     /* check whether we should intercept this file descriptor */
     if (unifyfs_intercept_fd(&fd)) {
+
+        LOGDBG("write %d bytes to fd %d", count, fd);
```

Comment:
style: swap these two lines, to kill the blank line after if() and then separate LOGDBG statement from following block of code